.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug Report
+about: To report a bug
+title: ""
+labels: 'bug'
+assignees: ''
+
+---
+
+## **Bug report**
+
+**Describe the bug**
+Provide a clear and concise description of what the bug is. 
+
+**Steps to reproduce the bug** 
+Steps to reproduce the behaviour:
+1. Go to '...'
+2. Tap on '....'
+3. Scroll down to '....'
+4. See '....'
+
+**Expected behaviour**
+Provide a clear and concise description of what you expected to happen.
+
+**Observed behaviour**
+Provide a clear and concise description of what actually happened.
+
+**Screenshots:** 
+Visual examples can help us better understand the issue.
+
+**Platform:** 
+What is the operating system your machine? What browser (Chrome, Internet Explorer, Firefox) did you use?
```

Comment:
I think keeping a bullet point structure like [here](https://github.com/MobilityData/gtfs-validator/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) would be helpful, so users can just copy paste their infos.


In our case, I think its reasonable to think we'd have:
- Browser version:
- OS version: